src/MDMMotionAnimator.h

```diff
@@ -103,4 +103,27 @@ NS_SWIFT_NAME(MotionAnimator)
                   keyPath:(nonnull MDMAnimatableKeyPath)keyPath
                completion:(nullable void(^)(void))completion;
 
+/**
+ Performs `animations` using the timing provided.
+
+ @param timing The timing to be used for the animation.
+ @param animations The block to be executed. Any animatable properties changed within this block
```

Comment:
Can we document the retain semantics of the block for callers? Is it as safe (unretained) as UIKit animations?

Reply:
Done.